package.json

```
@@ -25,29 +25,37 @@
     "lint": "eslint .",
     "pretest": "npm run lint",
     "test": "nyc mocha --async-only",
-    "test-types": "tsc -p test/types",
-    "azure-pipelines": "nyc mocha --async-only --reporter xunit -O output=test.xunit",
-    "coveralls": "nyc report --reporter=text-lcov | coveralls"
+    "test-types": "tsc -p test/types"
   },
   "dependencies": {
-    "end-of-stream": "^1.1.0",
-    "once": "^1.3.2",
-    "process-nextick-args": "^2.0.0",
-    "stream-exhaust": "^1.0.1"
+    "end-of-stream": "^1.4.4",
+    "once": "^1.4.0",
+    "process-nextick-args": "^2.0.1",
+    "stream-exhaust": "^1.0.2"
   },
   "devDependencies": {
-    "@types/node": "^9.3.0",
+    "@types/node": "^16.11.7",
     "coveralls": "github:phated/node-coveralls#2.x",
```

I believe we should remove this coveralls dep now
```suggestion
    "@types/node": "^16.11.7",
```